recipe/build.sh

```diff
@@ -11,7 +11,7 @@ then
     export LINKFLAGS="${LINKFLAGS} -stdlib=libc++ -std=c++11"
 fi
 
-./configure --prefix="${PREFIX}"
+./configure --prefix="${PREFIX}" --enable-unicode
```

Comment:
I think I just need pcre and not pcre2.  I think that pcre2 might break compatibility. It might make the most sense as its own feedstock.  But that isn't something that I need (yet).